web/server/codechecker_server/api/report_server.py

```diff
@@ -1777,6 +1779,17 @@ def _setReviewStatus(self, session, report_hash, status,
                                             review_status.date)
         session.add(system_comment)
 
+        # Confirmed, false positive and intentional reports are considered
```

Comment:
Is this statement still true?
https://github.com/Ericsson/codechecker/blob/93da64ffda93a6d098b8a3e182a69525ca47a52d/web/server/vue-cli/src/components/Statistics/Overview/Reports.vue#L22-L23

It would be good to update these hints so the user will know how the number are calculated.